### DISCUSSION TITLE
neodb_token/

### DISCUSSION THREAD

**Discussion by ghost** [eallion/eallion.com#97] [Giscus]
# neodb_token/

1. 注册 Neodb 账号注册 NeoDB 账号前，需要注册一个 Mastodon 长毛象宇宙的账号，有很多实例可以注册。然后用 Mastodon 账号就可以登录 NeoDB 了。最新的 NeoDB 似乎已经可以绑定邮箱登录了

https://www.eallion.com/neodb_token/

**Comment by benjamin603** [comment]
换评论了？之前记录都没了！

**Reply by eallion** [reply]
本来是需要关闭评论的，暂时留一个评论方式。

**Comment by lss53** [comment]
最后一步并没有返回access_token

**Reply by eallion** [reply]
现在可能 NeoDB 更新了，官方文档 https://neodb.social/developer/ 其实有 `How to authorize` 点进去就能看到完整的教程。

**Reply by ahjsrhj** [reply]
根据官方文档来 把最后一步URL改成了 `https://neodb.social/auth/oauth/token/` 也是不好使
返回404

**Reply by eallion** [reply]
我测试了一下，确实不能返回 Access Token 了，这个要等官方修复了。
